docs/tutorials/timeseries/forecasting-chronos.ipynb

```diff
@@ -9,13 +9,19 @@
     "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/autogluon/autogluon/blob/master/docs/tutorials/timeseries/forecasting-chronos.ipynb)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        time_limit=30,
</pre><p>Is this small time_limit on purpose? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/autogluon/autogluon/pull/4684/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Changed this to 60 seconds. My main goal here was to show that even with a short time limit we can some quite significant improvements in forecast accuracy.

So this is quite different from LLM fine-tuning where the entire process can take hours.